float64 range
float64 aoa
float64 velocity
float64 rcs
string[] target_ids
